### PR TITLE
[SC] Change bridge operator gas limit get,set logic into parent,child

### PR DESCRIPTION
**Pull request by hqjang-pepper**
## Proposed changes

- Seperated logic of getting and setting bridge operator's gas limit into child and parent operator.
- Change getter logic from method to property. So now we can call by parameter, like this.
```
> subbridge.ChildBridgeOperatorGasLimit
10000000
> subbridge.ParentBridgeOperatorGasLimit
10000000
```
issue: https://github.com/klaytn/klaytn/issues/1333

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: console/web3ext/web3ext.go [hunk 1]
```diff
@@ -1487,6 +1487,14 @@ web3._extend({
 			name: 'childOperatorFeePayer',
 			getter: 'subbridge_getChildOperatorFeePayer',
 		}),
+		new web3._extend.Property({
+			name: 'ParentBridgeOperatorGasLimit',
```
Comment (hyunsooda) [inline console/web3ext/web3ext.go:1491]: Could you follow the naming convention? All the other APIs start with lower case.

**Review by hyunsooda — COMMENTED** [review]
Could you apply these changes to the configuration files too?

**Comment by hqjang-pepper** [comment]
> Could you apply these changes to the configuration files too?

I think it's enough to have APIs. I'm not sure of the needs to put different initial gaslimit settings for each child/parent operators.

File: console/web3ext/web3ext.go (hunk 1 above)
Comment (hqjang-pepper) [reply console/web3ext/web3ext.go:1491]: I'll update the code. Thanks for the review.

**Comment by hyunsooda** [comment]
The current option that sets a given fee on both chains seems logically wrong. The parent/child chains are definitely separate networks, meaning having different chain settings. It must accompany the RPC call to set a different fee to the child or parent chain if the chain configuration is different on both chains. It's okay (but it seems not necessary) to have the current option that sets a given fee on both chains for the shortcut if we already have two separate options.

**Comment by hqjang-pepper** [comment]
I seperated `SC_DEFAULTGASLIMIT` into two(parent,child). Plz check.

File: cmd/utils/flaggroup.go [hunk 2]
```diff
@@ -288,7 +288,7 @@ var FlagGroups = []FlagGroup{
 			VTRecoveryIntervalFlag,
 			ServiceChainAnchoringFlag,
 			ServiceChainNewAccountFlag,
-			ServiceChainDefaultTxGasLimitFlag,
+			ServiceChainParentOperatorDefaultTxGasLimitFlag,
```
Comment (blukat29) [inline cmd/utils/flaggroup.go:291]: Isn't ServiceChainChildOperatorDefaultTxGasLimitFlag should be here too?

Comment (hqjang-pepper) [reply cmd/utils/flaggroup.go:291]: Oh I missed it. Thanks for your review.

File: build/packaging/linux/bin/kscnd [hunk 3]
```diff
@@ -308,8 +308,12 @@ start_node() {
         fi
     fi
 
-    if [ ! -z $SC_DEFAULTGASLIMIT ]; then
-        OPTIONS="$OPTIONS --scdefaultgaslimit $SC_DEFAULTGASLIMIT"
+    if [ ! -z $SC_PARENT_OPERATOR_DEFAULTGASLIMIT ]; then
```
Comment (hyunsooda) [inline build/packaging/linux/bin/kscnd:311]: `SC_PARENT_OPERATOR_DEFAULT_GAS_LIMIT` seems better.

File: build/packaging/linux/bin/kscnd [hunk 4]
```diff
@@ -308,8 +308,12 @@ start_node() {
...
+        OPTIONS="$OPTIONS --sc.parentop.defaultgaslimit $SC_PARENT_OPERATOR_DEFAULTGASLIMIT"
+    fi
+
+    if [ ! -z $SC_CHILD_OPERATOR_DEFAULTGASLIMIT ]; then
```
Comment (hyunsooda) [inline build/packaging/linux/bin/kscnd:315]: `SC_CHILD_OPERATOR_DEFAULT_GAS_LIMIT` seems better.

File: node/sc/bridge_accounts.go [hunk 5]
```diff
@@ -110,19 +110,28 @@ func (ba *BridgeAccounts) SetChildOperatorFeePayer(feePayer common.Address) erro
...
-func (ba *BridgeAccounts) GetBridgeOperatorGasLimit() uint64 {
-	return ba.pAccount.gasLimit
+// SetChildBridgeOperatorGasLimit changes GasLimit of child operator.
+func (ba *BridgeAccounts) SetChildBridgeOperatorGasLimit(fee uint64) {
+	ba.cAccount.gasLimit = fee
 }
 
 // NewBridgeAccounts returns bridgeAccounts created by main/service bridge account keys.
-func NewBridgeAccounts(am *accounts.Manager, dataDir string, db feePayerDB, gaslimit uint64) (*BridgeAccounts, error) {
+func NewBridgeAccounts(am *accounts.Manager, dataDir string, db feePayerDB, parentgaslimit, childgaslimit uint64) (*BridgeAccounts, error) {
```
Comment (hyunsooda) [inline node/sc/bridge_accounts.go:134]: `parentGasLimit` and `childGasLimit` seems better.

**Comment by hyunsooda** [comment]
I'm confused about whether the `Default` prefix is an important keyword in this context. The wording `default` can be interpreted as `pre-set` or `initial`. In my view, the `default` keyword does not have any semantics. It may be removed. Please add how do you think.

**Comment by hqjang-pepper** [comment]
> I'm confused about whether the `Default` prefix is an important keyword in this context. The wording `default` can be interpreted as `pre-set` or `initial`. In my view, the `default` keyword does not have any semantics. It may be removed. Please add how do you think.

I also think prefix `Default` is no need anymore. I'll delete it.

File: build/packaging/linux/bin/kscnd (hunk 3 above)
Comment (hqjang-pepper) [reply build/packaging/linux/bin/kscnd:311]: `gaslimit` should be single word. So I changed to `SC_PARENT_OPERATOR_GASLIMIT`.

File: build/packaging/linux/bin/kscnd [hunk 6]
```diff
@@ -308,12 +308,12 @@ start_node() {
         fi
     fi
 
-    if [ ! -z $SC_PARENT_OPERATOR_DEFAULTGASLIMIT ]; then
-        OPTIONS="$OPTIONS --sc.parentop.defaultgaslimit $SC_PARENT_OPERATOR_DEFAULTGASLIMIT"
+    if [ ! -z $SC_PARENT_OPERATOR_GASLIMIT ]; then
```
[the remark this answers is not in the document]
Comment (hqjang-pepper) [reply build/packaging/linux/bin/kscnd:311]: gaslimit should be single word. So I changed to SC_PARENT_OPERATOR_GASLIMIT.

File: build/packaging/linux/bin/kscnd [hunk 7]
```diff
@@ -308,12 +308,12 @@ start_node() {
...
+        OPTIONS="$OPTIONS --sc.parentop.gaslimit $SC_PARENT_OPERATOR_GASLIMIT"
     fi
 
-    if [ ! -z $SC_CHILD_OPERATOR_DEFAULTGASLIMIT ]; then
-        OPTIONS="$OPTIONS --sc.childop.defaultgaslimit $SC_CHILD_OPERATOR_DEFAULTGASLIMIT"
+    if [ ! -z $SC_CHILD_OPERATOR_GASLIMIT ]; then
```
[the remark this answers is not in the document]
Comment (hqjang-pepper) [reply build/packaging/linux/bin/kscnd:315]: gaslimit should be single word. So I changed to SC_PARENT_OPERATOR_GASLIMIT.

File: cmd/utils/flags.go [hunk 8]
```diff
@@ -743,12 +743,12 @@ var (
...
 	}
 	ServiceChainParentOperatorDefaultTxGasLimitFlag = cli.Uint64Flag{
-		Name:  "sc.parentop.defaultgaslimit",
+		Name:  "sc.parentop.gaslimit",
 		Usage: "Set the default value of gas limit for transactions made by bridge parent operator",
 		Value: 10000000,
 	}
 	ServiceChainChildOperatorDefaultTxGasLimitFlag = cli.Uint64Flag{
-		Name:  "sc.childop.defaultgaslimit",
+		Name:  "sc.childop.gaslimit",
```
[the remark this answers is not in the document]
Comment (hqjang-pepper) [reply cmd/utils/flags.go:751]: Thanks for the comment. I'll reflect it :)

File: node/sc/bridge_accounts.go [hunk 9]
```diff
@@ -131,7 +131,7 @@ func (ba *BridgeAccounts) SetChildBridgeOperatorGasLimit(fee uint64) {
 }
 
 // NewBridgeAccounts returns bridgeAccounts created by main/service bridge account keys.
-func NewBridgeAccounts(am *accounts.Manager, dataDir string, db feePayerDB, parentgaslimit, childgaslimit uint64) (*BridgeAccounts, error) {
+func NewBridgeAccounts(am *accounts.Manager, dataDir string, db feePayerDB, parentGaslimit, childGaslimit uint64) (*BridgeAccounts, error) {
```
[the remark this answers is not in the document]
Comment (hqjang-pepper) [reply node/sc/bridge_accounts.go:134]: I'll reflect it :)

File: node/sc/bridge_manager_test.go [hunk 10]
```diff
@@ -1249,7 +1249,7 @@ func TestErrorDuplicatedSetBridgeInfo(t *testing.T) {
 	config.DataDir = tempDir
 	config.VTRecovery = true
 
-	bacc, _ := NewBridgeAccounts(nil, tempDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), config.ServiceChainOperatorGasLimit)
+	bacc, _ := NewBridgeAccounts(nil, tempDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), config.ServiceChainParentOperatorGasLimit, config.ServiceChainChildOperatorGasLimit)
```
Comment (hyunsooda) [inline node/sc/bridge_manager_test.go:1252]:
```suggestion
	bacc, _ := NewBridgeAccounts(nil, tempDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), DefaultBridgeTxGasLimit, DefaultBridgeTxGasLimit)
```

File: node/sc/bridge_manager_test.go [hunk 11]
```diff
@@ -1316,7 +1316,7 @@ func TestScenarioSubUnsub(t *testing.T) {
...
```
Comment (hyunsooda) [inline node/sc/bridge_manager_test.go:1319]:
```suggestion
	bacc, _ := NewBridgeAccounts(nil, tempDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), DefaultBridgeTxGasLimit, DefaultBridgeTxGasLimit)
```

File: node/sc/bridge_manager_test.go [hunk 12]
```diff
@@ -1422,7 +1422,7 @@ func TestErrorDupSubscription(t *testing.T) {
...
```
Comment (hyunsooda) [inline node/sc/bridge_manager_test.go:1425]:
```suggestion
	bacc, _ := NewBridgeAccounts(nil, tempDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), DefaultBridgeTxGasLimit, DefaultBridgeTxGasLimit)
```

File: node/sc/bridge_manager_test.go [hunk 13]
```diff
@@ -1826,7 +1826,7 @@ func TestAnchoringPeriod(t *testing.T) {
 	config.DataDir = tempDir
 	config.VTRecovery = true
 
-	bAcc, _ := NewBridgeAccounts(nil, tempDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), config.ServiceChainOperatorGasLimit)
+	bAcc, _ := NewBridgeAccounts(nil, tempDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), config.ServiceChainParentOperatorGasLimit, config.ServiceChainChildOperatorGasLimit)
```
Comment (hyunsooda) [inline node/sc/bridge_manager_test.go:1829]:
```suggestion
	bAcc, _ := NewBridgeAccounts(nil, tempDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), DefaultBridgeTxGasLimit, DefaultBridgeTxGasLimit)
```

File: node/sc/bridge_manager_test.go [hunk 14]
```diff
@@ -1950,7 +1950,7 @@ func TestDecodingLegacyAnchoringTx(t *testing.T) {
...
```
Comment (hyunsooda) [inline node/sc/bridge_manager_test.go:1953]:
```suggestion
	bAcc, _ := NewBridgeAccounts(nil, tempDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), DefaultBridgeTxGasLimit, DefaultBridgeTxGasLimit)
```

File: node/sc/bridge_manager_test.go [hunk 15]
```diff
@@ -2058,7 +2058,7 @@ func TestGetBridgeContractBalance(t *testing.T) {
 	// Config Bridge Account Manager
 	config := &SCConfig{}
 	config.DataDir = tempDir
-	bacc, _ := NewBridgeAccounts(nil, config.DataDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), DefaultBridgeTxGasLimit)
+	bacc, _ := NewBridgeAccounts(nil, config.DataDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), config.ServiceChainParentOperatorGasLimit, config.ServiceChainChildOperatorGasLimit)
```
Comment (hyunsooda) [inline node/sc/bridge_manager_test.go:2061]:
```suggestion
	bacc, _ := NewBridgeAccounts(nil, config.DataDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), DefaultBridgeTxGasLimit, DefaultBridgeTxGasLimit)
```

File: node/sc/bridge_account_test.go [hunk 16]
```diff
@@ -164,7 +164,7 @@ func TestBridgeAccountInformation(t *testing.T) {
 	// Config Bridge Account Manager
 	config := &SCConfig{}
 	config.DataDir = tempDir
-	bAcc, err := NewBridgeAccounts(nil, config.DataDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), config.ServiceChainOperatorGasLimit)
+	bAcc, err := NewBridgeAccounts(nil, config.DataDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), config.ServiceChainParentOperatorGasLimit, config.ServiceChainChildOperatorGasLimit)
```
Comment (hyunsooda) [inline node/sc/bridge_account_test.go:167]:
```suggestion
	bAcc, err := NewBridgeAccounts(nil, config.DataDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), DefaultBridgeTxGasLimit, DefaultBridgeTxGasLimit)
```

File: node/sc/bridge_account_test.go [hunk 17]
```diff
@@ -43,7 +43,7 @@ func TestBridgeAccountLockUnlock(t *testing.T) {
...
```
Comment (hyunsooda) [inline node/sc/bridge_account_test.go:46]:
```suggestion
	bAcc, err := NewBridgeAccounts(nil, config.DataDir, database.NewDBManager(&database.DBConfig{DBType: database.MemoryDB}), DefaultBridgeTxGasLimit, DefaultBridgeTxGasLimit)
```

File: build/packaging/linux/bin/kscnd (hunk 6 above)
[the remark this answers is not in the document]
Comment (hyunsooda) [reply build/packaging/linux/bin/kscnd:311]: Okay.

File: build/packaging/linux/bin/kscnd (hunk 7 above)
[the remark this answers is not in the document]
Comment (hyunsooda) [reply build/packaging/linux/bin/kscnd:315]: Okay.

File: cmd/utils/nodecmd/nodeflags.go [hunk 18]
```diff
@@ -297,33 +299,8 @@ var KSENFlags = []cli.Flag{
...
-	utils.ChainDataFetcherKafkaReplicasFlag,
-	utils.ChainDataFetcherKafkaBrokersFlag,
-	utils.ChainDataFetcherKafkaPartitionsFlag,
-	utils.ChainDataFetcherKafkaTopicResourceFlag,
-	utils.ChainDataFetcherKafkaTopicEnvironmentFlag,
-	utils.ChainDataFetcherKafkaMaxMessageBytesFlag,
-	utils.ChainDataFetcherKafkaSegmentSizeBytesFlag,
-	utils.ChainDataFetcherKafkaRequiredAcksFlag,
-	utils.ChainDataFetcherKafkaMessageVersionFlag,
-	utils.ChainDataFetcherKafkaProducerIdFlag,
```
[the remark this answers is not in the document]
Comment (hqjang-pepper) [reply cmd/utils/nodecmd/nodeflags.go:326]: Oh I did mistake while solving conflict. Thanks for the catch.

**Review by hyunsooda — APPROVED** [review]

**Review by henry-will — CHANGES_REQUESTED** [review]

**Review by henry-will — APPROVED** [review]

**Review by hyunsooda — APPROVED** [review]

**Review by aidan-kwon — APPROVED** [review]

**Comment by hyunsooda** [comment]
Please close #1333 when you are done merging this PR.

**Review by blukat29 — APPROVED** [review]

**Review by sirano11 — APPROVED** [review]

**Pull request merged**